source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -1,147 +1,182 @@
 .. _placeholder-variable-text:
 
-===========================
-Placeholder (Variable) Text
-===========================
+============================
+Placeholder or Variable Text
+============================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+.. default-domain:: mongodb
 
-Placeholder text (also referred to as variable text or replaceable
-text) stands for an object whose specific name is unknown to us.
-Placeholders are included when documenting syntax for how a command or
-path should be constructed. Users supply the relevant value for the
+.. include:: /includes/styles/corrections.rst
+
+Use placeholder text stands for a value when you:
+
+- don't know its specific value
+- don't or shouldn't use example values
+
+You use placeholders when documenting syntax for how a command or path
+should be constructed. Users supply the relevant value for the
 placeholder when using the command or syntax.
 
+Placeholder text can also referred to as *variable text* or
+*replaceable text*.
+
 Placeholder text usually indicates the type of element that's being
 represented. For example, *directoryName* would likely indicate the
 name of a directory.
 
 .. note::
 
-   Placeholder text is distinct from *environment variables*.
-   Environment variables have established formats and names, such as
-   ``$account``, and their values are set in the system by users and
-   used consistently. By contrast, a placeholder is given a relevant
-   value by the user at the time that the user runs the code or types
-   the path. For information about formatting environment variables,
+   Placeholder text differs from *environment variables*. Environment
+   variables have established formats and names based upon the
+   applicable operating system. Users set their values in the system
+   and use them consistently. By contrast, a user gives the placeholder
+   a relevant value when the user runs the code or types the path.
+
+   To learn more about formatting environment variables,
    see :ref:`text-formatting`.
 
+Placeholder Text Guidelines
+---------------------------
+
 When creating placeholder text, use the following guidelines.
 
 .. note::
 
-   For specific information about showing placeholders for account
-   information such as account numbers, usernames, passwords, and API
-   keys, see :ref:`cloud-account-info`.
+   To learn about showing placeholders for API resources, see
+   :ref:`cloud-account-info`.
 
 .. list-table::
-   :widths: 50 50
+   :widths: 60 40
    :header-rows: 1
 
    * - Guidelines
 
      - Example
 
-   * - Within regular text, show placeholder text in italics.
+   * - Within regular, inline text, show placeholder text in italics
```

Comment:
Nitpick: this list is confusing since it only contains two items but the punctuation implies more than that. Suggestion:
"Within regular or inline text, show placeholder..."